test/kitchen/tasks/kitchen.py

```diff
@@ -0,0 +1,156 @@
+from invoke import task
+from invoke.exceptions import Exit
+
+import glob
+import json
+import os.path
+import re
+
+@task(iterable=['platlist'])
+def genconfig(
+    ctx,
+    platform=None,
+    provider=None,
+    osversions="all",
+    testfiles=None,
+    uservars=None,
+    platformfile="platforms.json",
+    platlist=None
+):
+    """
+    Create a kitchen config
+    """
+    if not platform and not platlist:
+        print("Must supply a platform to configure\n")
+        raise Exit(1)
+
+    if not testfiles:
+        print("Must supply one or more testfiles to include\n")
+        raise Exit(1)
```

Comment:
Since `testfiles` is always required, maybe we could make it a positional argument?